pkg/internal/vault/vault.go

```diff
@@ -30,7 +30,7 @@ import (
 	"github.com/hashicorp/vault/sdk/helper/certutil"
 	corelisters "k8s.io/client-go/listers/core/v1"
 
-	"github.com/jetstack/cert-manager/pkg/apis/certmanager/v1"
+	v1 "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1"
```

Comment:
we should rename this to `cmapi` if you mind changing that in the rest of the code too?

Reply:
code meaning file... not all of cert-manager